ngraph/core/include/openvino/core/descriptor/tensor.hpp

```diff
@@ -74,6 +76,15 @@ class OPENVINO_API Tensor {
     }
     size_t size() const;
 
+    using RTMap = std::map<std::string, std::shared_ptr<Variant>>;
```

Comment:
Please do not add new usings `RTMap` please use `RTMap` from `variant.hpp` instead

Reply:
fixed